isis/tests/OsirisRexOCamsCameraTests.cpp

```diff
@@ -95,3 +95,41 @@ TEST_F(OsirisRexCube, SamplingCam) {
   EXPECT_NEAR(cam->Line(), 5, 0.01);
   EXPECT_NEAR(cam->Sample(), 5, 0.01);
 }
+
+TEST_F(OsirisRexCube, PolyCamUpdatedIkCodes){
+setInstrument("-64500", "PolyCam");
+
+  OsirisRexOcamsCamera *cam = (OsirisRexOcamsCamera *)testCube->camera();
+
+  EXPECT_EQ(cam->instrumentRotation()->Frame(), -27002);
+
+  // Test kernel IDs
+  EXPECT_EQ(cam->CkFrameId(), -64000);
+  EXPECT_EQ(cam->CkReferenceId(), 1);
+  EXPECT_EQ(cam->SpkTargetId(), -64);
+  EXPECT_EQ(cam->SpkReferenceId(), 1);
```

Comment:
Probably worth checking the NaifIkCode in the Kernels group

Reply:
The camera doesn't set that value. That value is set by the ingestion program